fprettify/__init__.py

```diff
@@ -1556,11 +1570,18 @@ def get_arg_parser(args):
                             help="Overrides default fortran extensions recognized by --recursive. Repeat this option to specify more than one extension.")
         parser.add_argument('--version', action='version',
                             version='%(prog)s 0.3.6')
+        parser.add_argument('--where_forall_constructs', default=False,
```

Comment:
I'd suggest to use `--statement-constructs` or something similar as option name to indicate, that it handles hybrid language elements (those having the same form in statements and constructs) as constructs (as opposed to the default behavior which handles them as statements).